src/ngraph/rt_info.cpp

```diff
@@ -0,0 +1,80 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <ngraph/node.hpp>
+#include <ngraph/variant.hpp>
+
+#include "rt_info.hpp"
+
+namespace ngraph
+{
```

Comment:
Convention for cpp files is to `prepend ngraph::` to the name of the function and not wrap the whole file in the ngraph namespace. This saves one level of indenting.

Reply:
Done.